spring-cloud-kubernetes-test-support/src/main/java/org/springframework/cloud/kubernetes/integration/tests/commons/K8SUtils.java

```diff
@@ -218,7 +269,8 @@ public boolean isDeploymentReady(String deploymentName, String namespace) throws
 		}
 		V1Deployment deployment = deployments.getItems().get(0);
 		Integer availableReplicas = deployment.getStatus().getAvailableReplicas();
-		log.info("Available replicas for " + deploymentName + ": " + availableReplicas);
```

Comment:
I was really getting annoyed but that `null` for available replicas logs. not a big deal, but I feel better now